build.py

```diff
@@ -299,8 +299,12 @@ def build_package(target):
     else:
         os.chdir(os.path.join(build_dir,target))
 
+    number_of_jobs = get_jobs()
+    if target == 'elasticlient':
+        number_of_jobs = 1
+
```

Comment:
no need to do this - just don't use the `TEMPLATE_JOBS` when running `make` for elasticlient below